src/assets/scripts/client/ui/TrafficRateController.js

```diff
@@ -115,6 +120,7 @@ export default class TrafficRateController {
         this._onAirportChangeHandler = this.onAirportChange.bind(this);
         this._onChangeWindDirection = this.onChangeWindDirection.bind(this);
         this._onChangeWindSpeed = this.onChangeWindSpeed.bind(this);
+        this._onFormReset = this.onFormReset.bind(this);
```

Comment:
hmm, seems I've been sleeping on PRs in this file.  We typically add the word `handler` to the bound method. 
```js
this._onFormResetHandler = this.onFormReset.bind(this);
```

But I see the two methods above this do not follow this convention. I'd encourage you to make that change, but it isn't required.